test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -2205,4 +2240,36 @@ public static String resolveCustomDataPath(String index) {
     public static boolean inFipsJvm() {
         return Boolean.parseBoolean(System.getProperty(FIPS_SYSPROP));
     }
+
+    protected void awaitNoMoreSnapshotRunningOperations(String viaNode) throws Exception {
```

Comment:
NIT: reorder this to natural word order `awaitNoMoreRunningSnapshotOperations` :)